src/main/java/sk/hfa/projects/web/domain/responsebodies/InstagramRefreshTokenResource.java

```diff
@@ -0,0 +1,12 @@
+package sk.hfa.projects.web.domain.responsebodies;
+
+import lombok.Data;
+
+@Data
+public class InstagramRefreshTokenResource {
+
+    private String access_token;
+    private String token_type;
+    private int expires_in;
```

Comment:
Pre tieto properties to isté čo v prípade media_url.

Reply:
Sonar